sccp/figures/common.py

```diff
@@ -194,6 +194,42 @@ def plotProjs_SS(factors, projs, celltypeXA, color_palette, ax, size=100):
     sns.barplot(data=silhouetteDF.loc[silhouetteDF["Projs"] == 1], x="Cell Type", y = "Silhoutte Score", hue = "Cmp.", ax=ax[2*i + len(factors) + 2])
 
 
+
+def plotAllProjs_SS(projs, celltypeXA, color_palette, ax):
```

Comment:
@aarmey I am currently trying to replicate the process for measuring the SS, similar to `plotProjs_SS`, but do it for all conditions and see if cells from the same cell type behave similarly. However, I have been having trouble getting rid of cells that have 0's for the projections, since some cells might have values of 0's for once condition but not others.

Reply:
I don’t follow. Cells are not shared across conditions.